packages/perseus/src/widgets/interactive-graphs/interactive-graph-question-builder.ts

```diff
@@ -64,10 +64,7 @@ class InteractiveGraphQuestionBuilder {
                         labels: this.labels,
                         markings: this.markings,
                         range: [this.xRange, this.yRange],
-                        rulerLabel: "",
```

Comment:
This builder pattern is lovely because we don't have to go find out every place that we have test data and remove these. 💪